lisa/sut_orchestrator/azure/common.py

```diff
@@ -154,6 +155,12 @@
 METADATA_ENDPOINT = "http://169.254.169.254/metadata/instance?api-version=2021-02-01"
 
 
+class AzureVMStatus(Enum):
```

Comment:
It looks like the hibernation test case is Azure only now. Is the "deallocated" state common in other platforms? if so, it can be moved to common startstop feature.